Podfile

```diff
@@ -8,10 +8,9 @@ target 'NoteFind' do
   # Pods for NoteFind
 pod 'Parse'
 pod 'Parse/UI'
-pod 'DateTools'
-pod 'NotionClient'
-pod 'AppAuth'
 pod 'AFNetworking'
+pod 'Parse+NSCoding', '~> 0.1.4'
```

Comment:
Did you end up using this library? If not I'd remove it and rebuild your pods to avoid checking in more source files than you have to

Reply:
done!